draft-ietf-quic-transport.md

```diff
@@ -3246,7 +3246,21 @@ Implementors are encouraged to register version numbers of QUIC that they are
 using for private experimentation on the GitHub wiki at
 \<https://github.com/quicwg/base-drafts/wiki/QUIC-Versions\>.
 
-
+## Version Aliases
+
+In order to avoid ossification of the version number defined by this draft,
+servers announce a list of version numbers that they interpret as an alias for
+the version number used in this draft. Alias versions MUST NOT be a reserved
+version. Servers SHOULD send at least one version alias, and SHOULD frequently
+change the value that they announce. Each version alias contains a lifetime,
```

Comment:
If we have this, is there any point to the concept of reserved versions at all? Perhaps this depends on future version negotiation.

Reply:
This feature is being proposed to grease middleboxes; reserved versions exist to grease the peer endpoint.